bindings/ocaml/analysis/llvm_analysis.ml

```diff
@@ -20,3 +20,21 @@ external assert_valid_function : Llvm.llvalue -> unit
 external view_function_cfg : Llvm.llvalue -> unit = "llvm_view_function_cfg"
 external view_function_cfg_only : Llvm.llvalue -> unit
                                 = "llvm_view_function_cfg_only"
+
+external get_callgraph :  Llvm.llmodule -> Llvm.llcallgraph = "llvm_get_callgraph"
+external dispose_callgraph : Llvm.llcallgraph  -> unit = "llvm_dispose_callgraph"
+external get_callee_size : Llvm.llcallgraph -> Llvm.llvalue -> int = "llvm_get_callee_size"
+external get_callee_numrefs : Llvm.llcallgraph -> Llvm.llvalue -> int = "llvm_get_callee_num_refs"
+external get_ith_callee : Llvm.llcallgraph -> Llvm.llvalue -> int -> Llvm.llvalue option = "llvm_get_ith_callee"
+
+let fold_callees : callgraph:Llvm.llcallgraph -> init:'a -> f:('a -> Llvm.llvalue -> 'a) -> Llvm.llvalue -> 'a =
```

Comment:
I think that the existing llvm bindings are label-free, so better to keep them consistent. Also, better to mimic the arg order of other fold functions such as fold_successors